### PR TITLE
Update some metrics

### PR DESCRIPTION
**Pull request by SuperQ**
* Channel utilization Unit is percent.
* Drop "total" as it is the sum of transmit and receive.
* The metrics radio_transmit_{packets,retries} appear to gauges, not
counters.
* Convert Prometheus percents to ratio.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
`unifipoller_device_vap_data_mpdu_transmit_bytes_total` kinda looks like a counter, but sometimes jumps up in absolute value for a while and then goes back down.

File: pkg/promunifi/uap.go [hunk 1]
```diff
@@ -321,16 +319,15 @@ func (u *promUnifi) exportRadtable(r report, labels []string, rt unifi.RadioTabl
 				{u.UAP.RadioTxPower, prometheus.GaugeValue, t.TxPower, labelR},
 				{u.UAP.RadioAstBeXmit, prometheus.GaugeValue, t.AstBeXmit, labelR},
 				{u.UAP.RadioChannel, prometheus.GaugeValue, t.Channel, labelR},
-				{u.UAP.RadioCuSelfRx, prometheus.GaugeValue, t.CuSelfRx, labelR},
-				{u.UAP.RadioCuSelfTx, prometheus.GaugeValue, t.CuSelfTx, labelR},
-				{u.UAP.RadioCuTotal, prometheus.GaugeValue, t.CuTotal, labelR},
```
Comment (davidnewhall) [inline pkg/promunifi/uap.go]: Please fix the conflicts, I probably added more things with `_percent` too. Is `_ratio` preferred for percentage gauges?

File: pkg/promunifi/clients.go [hunk 2]
```diff
@@ -46,8 +46,8 @@ func descClient(ns string) *uclient {
 	return &uclient{
 		Anomalies:      prometheus.NewDesc(ns+"anomalies", "Client Anomalies", labelW, nil),
 		BytesR:         prometheus.NewDesc(ns+"transfer_rate_bytes", "Client Data Rate", labelW, nil),
-		CCQ:            prometheus.NewDesc(ns+"ccq_percent", "Client Connection Quality", labelW, nil),
-		Satisfaction:   prometheus.NewDesc(ns+"satisfaction_percent", "Client Satisfaction", labelW, nil),
+		CCQ:            prometheus.NewDesc(ns+"ccq_ratio", "Client Connection Quality", labelW, nil),
+		Satisfaction:   prometheus.NewDesc(ns+"satisfaction_ratoi", "Client Satisfaction", labelW, nil),
```
Comment (davidnewhall) [inline pkg/promunifi/clients.go]: `ratoi` - you made this typo 3 places.

File: pkg/promunifi/uap.go [hunk 3]
```diff
@@ -108,8 +107,8 @@ func descUAP(ns string) *uap {
 		VAPMacFilterRejections:   prometheus.NewDesc(ns+"vap_mac_filter_rejects_total", "VAP MAC Filter Rejections", labelV, nil),
 		VAPNumSatisfactionSta:    prometheus.NewDesc(ns+"vap_satisfaction_stations", "VAP Number Satisifaction Stations", labelV, nil),
 		VAPAvgClientSignal:       prometheus.NewDesc(ns+"vap_average_client_signal", "VAP Average Client Signal", labelV, nil),
-		VAPSatisfaction:          prometheus.NewDesc(ns+"vap_satisfaction_percent", "VAP Satisfaction", labelV, nil),
-		VAPSatisfactionNow:       prometheus.NewDesc(ns+"vap_satisfaction_now_percent", "VAP Satisfaction Now", labelV, nil),
+		VAPSatisfaction:          prometheus.NewDesc(ns+"vap_satisfaction_ratoi", "VAP Satisfaction", labelV, nil),
+		VAPSatisfactionNow:       prometheus.NewDesc(ns+"vap_satisfaction_now_ratoi", "VAP Satisfaction Now", labelV, nil),
```
Comment (davidnewhall) [inline pkg/promunifi/uap.go]:
```suggestion
		VAPSatisfactionNow:       prometheus.NewDesc(ns+"vap_satisfaction_now_ratio", "VAP Satisfaction Now", labelV, nil),
```

File: pkg/promunifi/uap.go [hunk 4]
```diff
@@ -108,8 +107,8 @@ func descUAP(ns string) *uap {
 		VAPMacFilterRejections:   prometheus.NewDesc(ns+"vap_mac_filter_rejects_total", "VAP MAC Filter Rejections", labelV, nil),
 		VAPNumSatisfactionSta:    prometheus.NewDesc(ns+"vap_satisfaction_stations", "VAP Number Satisifaction Stations", labelV, nil),
 		VAPAvgClientSignal:       prometheus.NewDesc(ns+"vap_average_client_signal", "VAP Average Client Signal", labelV, nil),
-		VAPSatisfaction:          prometheus.NewDesc(ns+"vap_satisfaction_percent", "VAP Satisfaction", labelV, nil),
-		VAPSatisfactionNow:       prometheus.NewDesc(ns+"vap_satisfaction_now_percent", "VAP Satisfaction Now", labelV, nil),
+		VAPSatisfaction:          prometheus.NewDesc(ns+"vap_satisfaction_ratoi", "VAP Satisfaction", labelV, nil),
```
Comment (davidnewhall) [inline pkg/promunifi/uap.go]:
```suggestion
		VAPSatisfaction:          prometheus.NewDesc(ns+"vap_satisfaction_ratio", "VAP Satisfaction", labelV, nil),
```

File: pkg/promunifi/usw.go [hunk 5]
```diff
@@ -78,7 +78,7 @@ func descUSW(ns string) *usw {
 		RxErrors:     prometheus.NewDesc(pns+"receive_errors_total", "Total Receive Errors", labelP, nil),
 		RxMulticast:  prometheus.NewDesc(pns+"receive_multicast_total", "Total Receive Multicast", labelP, nil),
 		RxPackets:    prometheus.NewDesc(pns+"receive_packets_total", "Total Receive Packets", labelP, nil),
-		Satisfaction: prometheus.NewDesc(pns+"satisfaction_percent", "Satisfaction", labelP, nil),
+		Satisfaction: prometheus.NewDesc(pns+"satisfaction_ratoi", "Satisfaction", labelP, nil),
```
Comment (davidnewhall) [inline pkg/promunifi/usw.go]:
```suggestion
		Satisfaction: prometheus.NewDesc(pns+"satisfaction_ratio", "Satisfaction", labelP, nil),
```

File: pkg/promunifi/uap.go (hunk 1 above)
Comment (davidnewhall) [reply pkg/promunifi/uap.go]: never mind, I'm impatient and want to get all this stuff "released" - I guess giving it time to be better is good too! Really appreciate your help!!

**Comment by davidnewhall** [comment]
Sorry to be in such a hurry on this one. I'm pretty much done with the dashboards, and I'm very pleased with how they turned out. Changing these metric names means I have to change the dashboards, so I want to keep up and keep moving!

**Comment by SuperQ** [comment]
No worries, I was already sleeping.

**Pull request merged**